src/transformers/integrations/peft.py

```diff
@@ -517,3 +517,59 @@ def _dispatch_accelerate_model(
             offload_dir=offload_folder,
             **dispatch_model_kwargs,
         )
+
+    def delete_adapters(self, adapter_names: Union[List[str], str]):
```

Comment:
Personally, I'd go with `delete_adapter`, even if the method allows to delete more than 1 adapter. There is a method in PEFT with the same name and in transformers, the PEFT methods also generally use singular form.

```suggestion
    def delete_adapter(self, adapter_names: Union[List[str], str]) -> None:
```